packages/contracts/deploy/new/30_plugins/00_plugin-setups/40_placeholder_setup.ts

```diff
@@ -27,3 +27,4 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
 export default func;
 func.tags = ['New', 'PlaceholderSetup'];
+func.skip = skipDeploy;
```

Comment:
The placeholder is actually a contract that we should deploy. Maybe not for GeoBrowser but for OSx.
This won't end up on the split-branch, right?

Reply:
I don't think Placeholder is used outside plugins. Since we don't deploy any plugins for Gii br,  we're good to skip this.